.changeset/metal-hornets-drum.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend-module-pinniped-provider': minor
```

Comment:
Maybe you can convince me otherwise, but I'm inclined not to call this a breaking change...
```suggestion
'@backstage/plugin-auth-backend-module-pinniped-provider': patch
```

Reply:
i agree, changed to patch.